### DISCUSSION TITLE
About Parameter Change in NAO.proto

### DISCUSSION THREAD

**Discussion by Xuanrrr** [cyberbotics/wrestling#62] [Q&A] [answered]
I would like to ask whether I can change the parameters in NAO.proto. For example, the lookupTable of the sonar?

**Comment by omichel** [comment, marked as answer]
Unfortunately, this is not possible. To be taken into account, it should be changed for all participants.
If you have a good reason for changing it we might consider changing it for all participants on the [competition repository](https://github.com/cyberbotics/wrestling/blob/main/protos/Nao.proto#L742-L765).
